.github/workflows/release.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   check:
     name: Check
     outputs:
-      version: steps.capture.outputs.version
+      version: ${{ steps.capture.outputs.version }}
```

Comment:
nit: this change should be back-ported to the prior commit, where the error was first introduced